stdlib/number.gr

```diff
@@ -108,6 +109,459 @@ export let mul = (*)
  */
 export let div = (/)
 
+/*
```

Comment:
We should note that this is based on their implementation, not that it's their code.

Reply:
@ospencer @spotandjake is that still the case if you've switch the algo to "Exponentiation by squaring" above?

Reply:
Yes, the squaring is just for the int^int case (and the algorithm used here is for all other cases).